tests/std/tests/strict_concepts_matrix.lst

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This file is identical to concepts_matrix.lst, except that all configurations include /permissive-.

```

Comment:
Please be aware of the :rotating_light: Stealth Merge Conflict :rotating_light: with #1150 adding `/Zc:noexceptTypes-`.

Reply:
thx @CaseyCarter